modules/ROOT/pages/database-administration/standard-databases/configuration-parameters.adoc

```diff
@@ -1,89 +1,6 @@
-:description: This page describes how to manage multiple active databases.
```

Comment:
should the page still have a description or is that no longer needed/wanted?

Reply:
What is the rule for when there is and isn't a page description? cause some files have one and some don't?

Reply:
All pages should have descriptions - good catch 👍

Reply:
I did look very quickly at some randomly picked files and some had one and some didn't so it's not just this file, might be good to go over all of the files and confirm if they have or don't have one :)